### ISSUE TITLE
error code *== (short long long)

### ISSUE THREAD

**Issue by hongtm77** [Ansem-SoD/Picofly#31]
Hi guys, i encountered an error that is not listed in the error code which is short, long, long (*==) any idea what is this mean?

**Comment by Quas7** [comment]
from the guide: [PicoFlyGuideV.6.2.pdf](https://github.com/Ansem-SoD/Picofly/blob/cc1099d6e9f659425b132fc73b9153fe07d605f8/PicoFlyGuideV.6.2.pdf)

```
** RST is not connected
*= CMD is not connected
=* D0 is not connected
== CLK is not connected
*** No eMMC CMD1 responce (bad eMMC?)
**= No eMMC block 1 read (should not happen)
*=* No eMMC block 0 read (eMMC init failure?)
*== No eMMC CMD1 request (poor wiring, or dead CPU)
=** eMMC init failure during glitch process
=*= CPU never reach BCT check, should not happen
==* CPU always reach BCT check (no glitch reaction, check
mosfet)
=== Glitch attempt limit reached, cannot glitch
=*** eMMC init failure
=**= eMMC write failure - comparison failed
=*=* eMMC write failure - write failed
=*== eMMC test failure - read failed
==** eMMC read failed during firmware update
==*= BCT copy failed - write failure
===* BCT copy failed - comparison failure
==== BCT copy failed - read failure
```

**Comment by shakangel** [comment]
Check the CLK cable. This error happened to me in an OLED where I had inavertandly compressed the CLK line with the APU heatsink. I guess the CLK line was shorting to ground or sending strange signals. Worth the check.

**Comment by lucasromeiro** [comment]
Sometimes I have this type of problem and I don't understand how to solve it!
The resistor solder at point A is perfect.... what else could cause this??

**Comment by kastras** [comment]
I had the same issue and the problem was the dat0 point, i had the flex touching two points. My switch can´t turn on with chip or without it. If you had the same issue, try removing the flex in dat0 and turning on the switch without chip, if it works its that, and you need to put again the flex in the correct possition.

**Comment by FabioKastro** [comment]
This is now happening to me, spent all day yesterday trying to figure out what was wrong, my Switch was working perfectly. Now that i was able to replicate the problem i found out that if you remove the SD Card while the emunand is booted the console will "crash??" and won't turn ON or show anything on the screen. If you force reboot by holding the power button the modchip will throw this error *== No eMMC CMD1 request (poor wiring, or dead CPU) and nothing on the screen, the switch appears to be completely dead. After resoldering everything step by step what seemed to work was resolder the flex on the CPU, the console was no longer dead and turned ON. 

BUT!!! Big BUT!!! 
Like i said, i was able to replicate this problem. Did the exact same, removed the SD card while booted into emunand and Switch no longer turns ON. So im pretty sure this was never a problem with soldering. The battery is now uplugged while i wait for everything to drain and hopefully fix the problem once again.

**Comment by TheCyberQuake** [comment]
Based on the finding in another open issue and my own experience, seems like the undocumented *== is usually tied to CLK due to improper handling of the error codes, where CLK isn't actually implemented. 
For OLED, the most common issue causing it seems to be when the user exposes the CLK point by scrapping the solder mask, they accidentally expose part of the surrounding ground plane. Pulling up the ribbon/cable, applying UV solder mask to any exposed ground plane surrounding it, and then soldering the ribbon back down seems to fix it in most cases.

**Comment by slewantoski** [comment]
I've had the exact same error code just now.
First could not find it. After some trying i realised that it only happens if i have the APU Shielding / Cooling attached.
Then found this post.

It was my CLK Line connected to the Shielding. Removed some of the tabs on this side and works perfectly now.

**Comment by TheCyberQuake** [comment]
> I've had the exact same error code just now.
> First could not find it. After some trying i realised that it only happens if i have the APU Shielding / Cooling attached.
> Then found this post.
> 
> It was my CLK Line connected to the Shielding. Removed some of the tabs on this side and works perfectly now.

Seems like we can confirm it's most often to show if CLK is shorted to ground then

**Comment by BagraNxt** [comment]
Found the solution if you encounter such problem. Just replace the modchip or buy an new one. Coz the problem is in the modchip.

**Comment by slewantoski** [comment]
Nope, my modchip still works without replacing the chip. I just removed a part from the APU shielding to remove the short to ground from CLK.
So it is not always the modchip

**Comment by TheCyberQuake** [comment]
> Found the solution if you encounter such problem. Just replace the modchip or buy an new one. Coz the problem is in the modchip.

Nope. Had several of us have already found that error code just means CLK is shorted to ground. 
Your replace fix was likely just the fact that you undid the old one with a bad solder and redid the new one, when you likely could have been fixed by just redoing your solder

**Comment by baalbeker** [comment]
Same issue happened to me after creating emunand partition that failed mid install.Wont boot.
Short long long yellow.
Little fed up at this point.

Doesnt work with another modchip,i had one spare.
Prolly will resolder it for the 3rd time poor me

**Comment by baalbeker** [comment]
It wont boot even after removing the chip so i assume cpu is dead

**Comment by TheCyberQuake** [comment]
> It wont boot even after removing the chip so i assume cpu is dead

Sounds like you need to go back and make sure you didn't short anything.
Depending on which system, there are plenty of places that you wouldn't be able to tell it's shorted without microscope, and then with it can be difficult.
And if you did short it out, that would be due to the install

**Comment by baalbeker** [comment]
Checked with multimeter and it looks like there is short on both ceramic capacitors(sp1/2).
Will replacing them with new fix it?
Should i check something else as well ?

**Comment by TheCyberQuake** [comment]
> Checked with multimeter and it looks like there is short on both ceramic capacitors(sp1/2).
> Will replacing them with new fix it?
> Should i check something else as well ?

This isn't tech support for damaged hardware. This isn't a forum, its a bug report. Search for help online as it's no longer related to the project here

**Comment by treyus30** [comment]
> > Found the solution if you encounter such problem. Just replace the modchip or buy an new one. Coz the problem is in the modchip.
> 
> Nope. Had several of us have already found that error code just means CLK is shorted to ground. Your replace fix was likely just the fact that you undid the old one with a bad solder and redid the new one, when you likely could have been fixed by just redoing your solder

I'm getting this error but the console boots to the normal screen. Wouldn't a short to ground prevent boot completely?

**Comment by dannn413** [comment]
> confirm

What could it be if its happening with on a v1 with a core style chip that the eMMC plugs into?

**Comment by tired9494** [comment]
> > > Found the solution if you encounter such problem. Just replace the modchip or buy an new one. Coz the problem is in the modchip.
> > 
> > 
> > Nope. Had several of us have already found that error code just means CLK is shorted to ground. Your replace fix was likely just the fact that you undid the old one with a bad solder and redid the new one, when you likely could have been fixed by just redoing your solder
> 
> I'm getting this error but the console boots to the normal screen. Wouldn't a short to ground prevent boot completely?

I'm also getting this error but boot to normal system. Did you find out what caused it?

**Comment by treyus30** [comment]
> > > > Found the solution if you encounter such problem. Just replace the modchip or buy an new one. Coz the problem is in the modchip.
> > > 
> > > 
> > > Nope. Had several of us have already found that error code just means CLK is shorted to ground. Your replace fix was likely just the fact that you undid the old one with a bad solder and redid the new one, when you likely could have been fixed by just redoing your solder
> > 
> > 
> > I'm getting this error but the console boots to the normal screen. Wouldn't a short to ground prevent boot completely?
> 
> I'm also getting this error but boot to normal system. Did you find out what caused it?

I had a loose pin. Re-soldered and it eventually worked. Wasn't really documented anywhere.
